lib/Sema/CSApply.cpp

```diff
@@ -5399,10 +5399,15 @@ Expr *ExprRewriter::coerceCallArguments(Expr *arg, AnyFunctionType *funcType,
 
       // Otherwise, create a call of the default argument generator.
       } else {
-        defArg =
-            new (tc.Context) DefaultArgumentExpr(callee, paramIdx,
-                                                 arg->getStartLoc(),
-                                                 param.getParameterType());
+        const ParamDecl *defaultParam = getParameterAt(cast<ValueDecl>(callee.getDecl()), newArgs.size());
+        defArg = defaultParam->getDefaultValue();
+
+        if (defArg == nullptr) {
```

Comment:
It looks like defArg has to be nullptr at this point, because we're in the else branch of an `if (defArg)` check.

Reply:
`defArg` is set two lines above (line 5404).

Reply:
D'oh! My apologies.

Reply:
No worries :) Could you trigger the bot?

Reply:
There's a merge conflict on your PR